frontend/hub/administration/repositories/RepositoryPage/RepositoryDistributions.tsx

```diff
@@ -0,0 +1,71 @@
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useOutletContext } from 'react-router-dom';
+import { CopyCell, ITableColumn, PageTable } from '../../../../../framework';
+import { pulpAPI } from '../../../common/api/formatPath';
+import { useHubView } from '../../../common/useHubView';
+import { Repository } from '../Repository';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+import { Distribution } from '../../../collections/UploadCollection';
+
+export function RepositoryDistributions() {
+  const { t } = useTranslation();
+  const { repository } = useOutletContext<{ repository: Repository }>();
+
+  const view = useHubView<Distribution>({
+    url: pulpAPI`/distributions/ansible/ansible/?repository=${repository.pulp_href}`,
+    keyFn: (distro) => distro.name,
+    defaultSort: 'name',
+  });
+
+  const tableColumns = useRepositoryDistributionColumns();
+
+  return (
+    <PageTable<Distribution>
+      id="hub-repository-distributions-table"
+      tableColumns={tableColumns}
+      errorStateTitle={t('No distributions')}
+      emptyStateTitle={t('No distributions yet')}
+      emptyStateDescription={t('You can edit this repository to create a distribution.')}
+      emptyStateButtonIcon={<PlusCircleIcon />}
+      {...view}
+      defaultTableView="table"
+      defaultSubtitle={t('Distribution')}
+    />
+  );
+}
+
+export function useRepositoryDistributionColumns() {
+  const { t } = useTranslation();
+
+  return useMemo<ITableColumn<Distribution>[]>(
+    () => [
+      {
+        header: t('Name'),
+        type: 'text',
+        value: (distribution: Distribution) => distribution.name,
+        sort: 'name',
+      },
+      {
+        header: t('Base path'),
+        type: 'text',
+        value: (distribution: Distribution) => distribution.base_path,
+        sort: 'base_path',
+      },
+      {
+        header: t('Created date'),
```

Comment:
```suggestion
        header: t('Created'),
```

(because https://github.com/ansible/ansible-ui/pull/2830/files)

Reply:
Fixed :)